Resources/config/yui.yml

```diff
@@ -1327,3 +1329,18 @@ system:
                         - 'ez-viewservicebaseplugin'
                         - 'ez-pluginregistry'
                     path: %ez_platformui.public_dir%/js/views/services/plugins/ez-locationswapplugin.js
+                ez-searchview:
+                    requires: ['ez-templatebasedview', 'ez-searchlistview', 'searchview-ez-template', 'ez-loadmorepagination']
```

Comment:
ez-loadmorepagination not needed and as you pointed out in https://github.com/ezsystems/PlatformUIBundle/pull/659#discussion_r76408527, node-style should be added here